src/compas_view2/objects/polylineobject.py

```diff
@@ -26,9 +27,13 @@ def _lines_data(self):
         positions = []
         colors = []
         elements = []
-        for i in range(len(polyline.points)-1):
+        if self.closed:
+            indexes = range(len(polyline.points))
+        else:
+            indexes = range(len(polyline.points)-1)
+        for i in indexes:
```

Comment:
why not go through the indices in pairs using `pairwise`? seems to remove the need for the modulo operation...

Reply:
makes sense!